metaworld/envs/mujoco/sawyer_xyz/sawyer_peg_insertion_side_v2.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
+from gym.spaces import Box
+
+from metaworld.envs.env_util import get_asset_full_path
+from metaworld.envs.mujoco.sawyer_xyz.base import SawyerXYZEnv
+
+
+class SawyerPegInsertionSideEnvV2(SawyerXYZEnv):
+    """
+    Motivation for V2:
+        V1 was completely unsolvable because the observation didn't say where
```

Comment:
Please avoid language which is this strong. it's not completely unsolvable, but it is very difficult to solve.